res/values-ca/strings.xml

```diff
@@ -1024,234 +1024,234 @@
+    <string name="ALLY">ALIAT</string>
+    <string name="Rename">Reanomenar</string>
+    <string name="Set_friendly_name_">Estableix un nom amigable.</string>
+    <string name="Leave_blank_for_none_">Deixeu-ne en blanc per a cap.</string>
+    <string name="Vote_Kick">Expulsar jugador</string>
     <string name="FONT">FONT</string>
     <string name="CIRCLE">CIRCLE</string>
-    <string name="SQUARES">SQUARES</string>
-    <string name="TRICK_MODE">TRICK MODE</string>
+    <string name="SQUARES">QUADRÍCULA</string>
+    <string name="TRICK_MODE">MODE TRICK</string>
     <string name="ICE_SPLIT">ICE SPLIT</string>
     <string name="TRICK_SPLIT">TRICK SPLIT</string>
     <string name="POP_SPLIT">POP SPLIT</string>
     <string name="PUSH_SPLIT">PUSH SPLIT</string>
-    <string name="Level_Colors">Level Colors</string>
-    <string name="click_info">Autoclick / Ultraclick do not work in FFA Classic!</string>
-    <string name="Hooked">Hooked</string>
-    <string name="Be_pulled_by_a_hookshot_spell_for_10_seconds">Be pulled by a hookshot spell for 10 seconds</string>
+    <string name="Level_Colors">Colors de nivell</string>
+    <string name="click_info">Autoclick / Ultraclick no funcionen a TCT Classic!</string>
+    <string name="Hooked">Enganxat</string>
+    <string name="Be_pulled_by_a_hookshot_spell_for_10_seconds">Deixa't tirar per un encanteri de ganxo durant 10 segons</string>
     <string name="Pack">Pack</string>
-    <string name="Skin_Pack">Skin Pack</string>
-    <string name="SKIN_PACK">SKIN PACK</string>
+    <string name="Skin_Pack">Pack de skin</string>
+    <string name="SKIN_PACK">PACK DE SKIN</string>
     <string name="HALLOWEEN">HALLOWEEN</string>
 
-    <string name="pack_info">After you purchase this skin pack your exclusive skins will appear at the top of the skin selection menus.</string>
-
-    <string name="EXCLUSIVE_skins_pets_hats_and_more_">EXCLUSIVE Skins, Pets, Hats and More!</string>
-    <string name="BUY_NOW_">BUY NOW!</string>
-    <string name="already_purchased">You or the person you are buying this for already own this.</string>
-    <string name="Halo_Animations">Halo Animations</string>
-    <string name="Trick_Notifications">Trick Notifications</string>
-    <string name="Channel">Channel</string>
-    <string name="Choose_Text_Size">Choose Text Size</string>
-    <string name="LEVEL_COLOR">LEVEL COLOR</string>
-    <string name="INITIATE">INITIATE</string>
-    <string name="x3_Clan_XP_Perm">3x Clan XP Permanent!</string>
-    <string name="x2_Clan_XP_Perm">2x Clan XP Permanent!</string>
-    <string name="Japan">Japan</string>
-    <string name="South_Korea">South Korea</string>
+    <string name="pack_info">Després de comprar aquest paquet d'aspectes, les vostres pells exclusives apareixeran a la part superior dels menús de selecció d'aspectes.</string>
+
+    <string name="EXCLUSIVE_skins_pets_hats_and_more_">Pells, mascotes, barrets i més EXCLUSIVES!</string>
```

Comment:
I've reviewed again and found some additional inaccuracies: "skin" should not be literally translated. Please make sure you fixed them. About the line 1078, the structure order needs a minor fix.

```suggestion
    <string name="pack_info">Després de comprar aquest paquet de skins, les vostres skins exclusives apareixeran a la part superior dels menús de selecció de skins.</string>

    <string name="EXCLUSIVE_skins_pets_hats_and_more_">Skins, mascotes, barrets EXCLUSIVES i més!</string>